### ISSUE TITLE
New translation: Hindi [hi]

### ISSUE THREAD

**Issue by nikramakrishnan** [squidfunk/mkdocs-material#803]
## Instructions

1. Check, if your language is already available: [here](http://bit.ly/2DCzaL0)
2. If it isn't, please translate the labels on the right:

``` jinja
{% macro t(key) %}{{ {
  "language": "hi",
  "clipboard.copy": "क्लिपबोर्ड पर कॉपी करें",
  "clipboard.copied": "क्लिपबोर्ड पर कॉपी कर दिया गया",
  "edit.link.title": "इस पृष्ठ को संपादित करें",
  "footer.previous": "पिछला",
  "footer.next": "आगामी",
  "meta.comments": "टिप्पणियाँ",
  "meta.source": "स्रोत",
  "search.language": "hi",
  "search.placeholder": "खोज",
  "search.result.placeholder": "खोज शुरू करने के लिए टाइप करें",
  "search.result.none": "कोई मिलान डॉक्यूमेंट नहीं",
  "search.result.one": "1 मिलान डॉक्यूमेंट",
  "search.result.other": "# मिलान डाक्यूमेंट्स",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "विषय पर बढ़ें",
  "source.link.title": "रिपॉजिटरी पर जाएं",
  "toc.title": "विषय - सूची"
}[key] }}{% endmacro %}
```

> Thanks!

:-)

**Comment by squidfunk** [comment]
Awesome, thanks! Is Hindi (Hindu?) RTL or LTR?

**Comment by nikramakrishnan** [comment]
It's Hindi, and is LTR, like every other Indian language. 😄

**Comment by squidfunk** [comment]
Released as part of 2.9.2. Thanks for your contribution!
